tests/core_tests.py

```diff
@@ -503,7 +533,8 @@ def test_dashboard_with_created_by_can_be_accessed_by_public_users(self):
         )
         self.grant_public_access_to_table(table)
 
-        dash = db.session.query(models.Dashboard).filter_by(dashboard_title="Births").first()
+        dash = db.session.query(models.Dashboard).filter_by(
```

Comment:
nit: I've been breaking long sqlalchemy queries method-chaining-style, but in python it needs parenthesis as in:
```
dash = (
    db.session.query(models.Dashboard)
    .filter_by(slug="births")
    .first()
)
```